docs/sops/msft-test-test-tenant-access.md

```diff
@@ -0,0 +1,284 @@
+
+> **⚠️ IMPORTANT: Service Principal Credential Expires on September 30, 2026**
+> 
+> The OpenShift CI service principal credential must be rotated before this date to prevent CI/CD pipeline failures.
+> 
+> **To rotate credentials:**
+> - See instructions: [`dev-infrastructure/openshift-ci/README.md`](../../dev-infrastructure/openshift-ci/README.md)
+> - Run script: [`dev-infrastructure/openshift-ci/recycle-openshift-release-bot-creds.sh`](../../dev-infrastructure/openshift-ci/recycle-openshift-release-bot-creds.sh)
+
+
+## Common Tasks
+
+### Requesting Quota Increases
+
+#### Current Quota Allocations
+
+The following quotas have been requested and approved for ARO HCP E2E testing:
+
+**STAGE Environment (uksouth):**
+| Quota Type | Current Allocation | Justification |
+|------------|-------------------|---------------|
+| Public IP Addresses | 300 | Support 15 parallel tests × 2 IPs per test |
+| Total Regional vCPUs | 300 | Support 12 parallel nodepools × 16 vCPUs per nodepool |
+| Standard DSv3 Family vCPUs | 300 | Node pool VMs use Standard_D8s_v3 (8 cores each) |
+
+**PRODUCTION Environment (8 Regions):**
+
+*Public IP Addresses (All 8 regions):*
+| Region | Public IP Limit |
+|--------|----------------|
+| uksouth | 300 |
+| brazilsouth | 300 |
+| centralindia | 300 |
+| switzerlandnorth | 300 |
+| canadacentral | 300 |
+| australiaeast | 300 |
+| westeurope | 300 |
+| eastus2 | 300 |
+
+*vCPU Quotas (Active E2E Test Regions):*
+| Region | DSv3 vCPUs |
+|--------|------------|
+| uksouth | 300 |
+| brazilsouth | 300(pending) |
+| centralindia | 300 |
+
+**Note**: Other 5 production regions (switzerlandnorth, canadacentral, australiaeast, westeurope, eastus2) do not have E2E tests configured yet, so vCPU quotas were not requested.
+
+#### How to Request Additional Quotas
+
+If you need to request quota increases for new regions or resources:
+
+1. Navigate to [Azure Portal](https://portal.azure.com)
+2. Select **Subscriptions** → **ARO HCP E2E - Staging** (or **ARO HCP E2E** for prod)
+3. In the left menu, select **Usage + quotas**
+4. Find the quota you need to increase (e.g., "Total Regional vCPUs")
+5. Click the pencil icon and request a new limit
+
+### Disabling Entra ID Soft Deletion
+
+High-volume E2E testing can hit the Entra ID directory object limit (500,000) due to soft-deleted objects counting towards the quota. We are working on the request of disabling Entra ID Soft Deletion
+
+
+### Cleaning Up Test Resources
+
+E2E tests should clean up after themselves, but sometimes resources are left behind:
+
+```bash
+# List resource groups created by E2E tests
+az group list --query "[?starts_with(name, 'premerge-') || starts_with(name, 'periodic-')].name" -o table
+
+# Delete old resource groups (use with caution!)
+az group delete --name <resource-group-name> --yes --no-wait
+```
+
+**⚠️ Warning**: Only delete resources you created or that are confirmed to be expired test resources.
+
+## Automated Cleanup
```

Comment:
Wait, if this exists, just delete the above section & don't invite anyone to do manual cleanups. Make the automated deletion _aggressive_